api/egl.xml

```diff
@@ -886,7 +886,8 @@
         <enum value="0x3376" name="EGL_STREAM_IMAGE_AVAILABLE_NV"/>
         <enum value="0x3377" name="EGL_DRM_RENDER_NODE_FILE_EXT"/>
         <enum value="0x3378" name="EGL_STREAM_CONSUMER_IMAGE_USE_SCANOUT_NV" />
-            <unused start="0x3379" end="0x339F"/>
+        <enum value="0x337A" name="EGL_ALLOC_NEW_DISPLAY_EXT"/>
```

Comment:
337A is your unused, 3379 is what you have in the extension spec below.

Reply:
Agh, sorry about that. Fixed.